submissions/AndriiZahumnyi/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+                    <p>Calendar</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/mail.png" alt="mail icon">
+                    <p>Mail</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/news.png" alt="news icon">
+                    <p>News</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/maps.png" alt="maps icon">
+                    <p>Maps</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/files.png" alt="files icon">
+                    <p>Files</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/contacts.png" alt="contacts icon">
+                    <p>Contacts</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/translate.png" alt="translate icon">
+                    <p>Translate</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/videos.png" alt="videos icon">
+                    <p>Videos</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/search.png" alt="search icon">
+                    <p>Search</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/photos.png" alt="photos icon">
+                    <p>Photos</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/apps.png" alt="apps icon">
+                    <p>Apps</p>
+                  </div>
+                </a>
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/documents.png" alt="documents icon">
+                    <p>Documents</p>
+                  </div>
+                </a>
+              </div>
+              <input id="check02" type="checkbox" name="more_menu" />
+              <div class="more_sub_menu">
+                <hr />
```

Comment:
I recommend use pseudo-element `:before` or `:after` if you want to add line, but in this case you are free in your choise